doc/MediaSigning.xml

```diff
@@ -0,0 +1,689 @@
+                  <entry><para>Every document</para></entry>
+                  <entry><para>General info like GOP number</para></entry>
+                </row>
+                <row>
+                  <entry><para>Hash list</para></entry>
+                  <entry><para>2</para></entry>
+                  <entry><para>Every document</para></entry>
+                  <entry><para>List of frame hashes</para></entry>
+                </row>
+                <row>
+                  <entry><para>Signature</para></entry>
+                  <entry><para>3</para></entry>
+                  <entry><para>Every document</para></entry>
+                  <entry><para>The signature of the document and signing algorithm</para></entry>
+                </row>
+                <row>
+                  <entry><para>Public key</para></entry>
+                  <entry><para>4</para></entry>
+                  <entry><para>Every n:th document (or once?)</para></entry>
+                  <entry><para>The public signing key</para></entry>
+                </row>
+                <row>
+                  <entry><para>ONVIF vendor info</para></entry>
+                  <entry><para>5</para></entry>
+                  <entry><para>Every n:th document (or once?)</para></entry>
+                  <entry><para>Information about the device and vendor</para></entry>
+                </row>
+                <row>
+                  <entry><para>Certificate</para></entry>
+                  <entry><para>6</para></entry>
+                  <entry><para>Once</para></entry>
+                  <entry><para>The device video signing key certificate and the intermediate
+                    CA signing certificate.</para></entry>
+                </row>
+                <row>
+                  <entry><para>Arbitrary data</para></entry>
+                  <entry>
+                    <para>7</para>
+                  </entry>
+                  <entry><para>Optional</para></entry>
+                  <entry><para>You put your weed in there</para></entry>
+                </row>
+              </tbody>
+            </tgroup>
+          </table>
+        </para>
+      </section>
+      <section>
+        <title>General GOP info</title>
+        <para>Includes GOP specific information like a GOP counter and a timestamp of the first frame.</para>
+        <itemizedlist>
+          <listitem>
+            <para>Tag version (1 byte)</para>
+          </listitem>
+          <listitem>
+            <para>GOP counter (4 bytes)</para>
+          </listitem>
+          <listitem>
+            <para>ONVIF Signed Media version (X bytes)</para>
+            <para>This will prevent version mismatch between device and client.</para>
+          </listitem>
+          <listitem>
+            <para>The UTC (8 bytes) based time represented by the number of 100-nanosecond intervals since
+              January 1, 1601 of the first media sample in the first fragment.</para>
```

Comment:
So long as the systems are using a 64bit word to represent Epoch time in their systems implementations.

Reply:
Yes 64 bit